### ISSUE TITLE
500 Error for occurrence view - missing assertion codes

### ISSUE THREAD

**Issue by djtfmartin** [AtlasOfLivingAustralia/la-pipelines#387]
// 20210531140555
// https://biocache-test.ala.org.au/ws/occurrence/ffe2ab71-42e2-44ff-86d8-a80ded33cce9

```
{
  "message": "No value present",
  "errorType": "Server error",
  "statusCode": 500
}

```

**Comment by djtfmartin** [comment]
Fixed in 3.0.9

**Comment by nickdos** [comment]
Link in description shows expected JSON data ✅
